till_tech/www/lib/ionic/scss/_util.scss

```diff
@@ -0,0 +1,296 @@
+  padding-right: $content-padding;
+}
+
+.padding-bottom,
+.padding-vertical {
+  padding-bottom: $content-padding;
+}
+
+.padding-left,
+.padding-horizontal {
+  padding-left: $content-padding;
+}
+
+
+/**
+ * Scrollable iFrames
+ * --------------------------------------------------
+ */
+
+.iframe-wrapper {
+  position: fixed;
+  -webkit-overflow-scrolling: touch;
+  overflow: scroll;
+
+  iframe {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+
+/**
+ * Rounded
+ * --------------------------------------------------
+ */
+
+.rounded {
+  border-radius: $border-radius-base;
+}
+
+
+/**
+ * Utility Colors
+ * --------------------------------------------------
+ * Utility colors are added to help set a naming convention. You'll
+ * notice we purposely do not use words like "red" or "blue", but
+ * instead have colors which represent an emotion or generic theme.
+ */
+
+.light, a.light {
+  color: $light;
+}
+.light-bg {
+  background-color: $light;
+}
+.light-border {
+  border-color: $button-light-border;
+}
+
+.stable, a.stable {
+  color: $stable;
+}
+.stable-bg {
+  background-color: $stable;
+}
+.stable-border {
+  border-color: $button-stable-border;
+}
+
+.positive, a.positive {
+  color: $positive;
+}
+.positive-bg {
+  background-color: $positive;
+}
+.positive-border {
+  border-color: $button-positive-border;
+}
+
+.calm, a.calm {
+  color: $calm;
+}
+.calm-bg {
+  background-color: $calm;
+}
+.calm-border {
+  border-color: $button-calm-border;
+}
+
+.assertive, a.assertive {
+  color: $assertive;
+}
+.assertive-bg {
+  background-color: $assertive;
+}
+.assertive-border {
+  border-color: $button-assertive-border;
+}
+
+.balanced, a.balanced {
+  color: $balanced;
+}
+.balanced-bg {
+  background-color: $balanced;
+}
+.balanced-border {
+  border-color: $button-balanced-border;
+}
+
+.energized, a.energized {
+  color: $energized;
+}
+.energized-bg {
+  background-color: $energized;
+}
+.energized-border {
+  border-color: $button-energized-border;
+}
+
+.royal, a.royal {
+  color: $royal;
+}
+.royal-bg {
+  background-color: $royal;
+}
+.royal-border {
+  border-color: $button-royal-border;
+}
+
+.dark, a.dark {
+  color: $dark;
+}
+.dark-bg {
+  background-color: $dark;
+}
+.dark-border {
+  border-color: $button-dark-border;
+}
+
+[collection-repeat] {
+  /* Position is set by transforms */
+  left: 0 !important;
+  top: 0 !important;
+  position: absolute !important;
+  z-index: 1;
+}
+.collection-repeat-container {
+  position: relative;
+  z-index: 1; //make sure it's above the after-container
+}
+.collection-repeat-after-container {
+  z-index: 0;
+  display: block;
+
+  /* when scrolling horizontally, make sure the after container doesn't take up 100% width */
+  &.horizontal {
+    display: inline-block;
+  }
+}
+
+// ng-show fix for windows phone
+// https://www.hoessl.eu/2014/12/on-using-the-ionic-framework-for-windows-phone-8-1-apps/
+[ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak,
```

Comment:
Selector `ng\:cloak` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line